homeassistant/components/rejseplanen/sensor.py

```diff
@@ -157,24 +163,15 @@ def __init__(self, stop_id, route, direction, departure_type):
         self.route = route
         self.direction = direction
         self.departure_type = departure_type
-        self.info = self.empty_result()
-
-    def empty_result(self):
-        """Object returned when no departures are found."""
-        return [{ATTR_DUE_IN: 'n/a',
-                 ATTR_DUE_AT: 'n/a',
-                 ATTR_TYPE: 'n/a',
-                 ATTR_ROUTE: self.route,
-                 ATTR_DIRECTION: 'n/a',
-                 ATTR_STOP_NAME: 'n/a'}]
+        self.info = []
 
     def update(self):
         """Get the latest data from rejseplanen."""
         import rjpl
         self.info = []
 
         def intersection(lst1, lst2):
-            """Return items contained in both lists."""
+            """Return items contained in both lists"""
```

Comment:
Docstrings should end with period.

Reply:
Indeed. Didn't see that sneak in there.

Reply:
Oh, as CoMPaTech asked on Discord, should the reviewer or the author resolve conversations?

Reply:
We don't have specific guidelines on that. I suggest you resolve when you think it's resolved. That will make the PR history more clean quicker. The reviewer can unresolve as needed.